cronos-play/getting-started_unreal/quick-start/erc1155.md

```diff
@@ -8,7 +8,7 @@ All ERC1155 functions are members of **DefiWalletCoreActor**. The Target should
 
 Get erc-1155 balance
 
-<figure><img src="../../../.gitbook/assets/image (6).png" alt=""><figcaption></figcaption></figure>
+<figure><img src="../../../.gitbook/assets/image (6) (3).png" alt=""><figcaption></figcaption></figure>
```

Comment:
maybe add some image verbosity instead of (number)?

Reply:
ok

Reply:
Images are from clipboard, the number were assigned by gitbook. Let me check if I can name the image.

Reply:
Renamed.